src/CertifiedData.mo

```diff
@@ -0,0 +1,33 @@
+/**
+
+Certified data
+
+The Internet Computer allows canister to store a small amount of data during
+update method processing so that during query call processing, they can obtain
+a certificate about that data.
+
+This module provides a _low-level_ interface to this API, aimed at advanced
+users and library implementors. See the Internet Computer Functional
```

Comment:
Can we insert a link here or are we still unpublished?

Reply:
Still unpublished unfortunatel